transmart-rest-api/src/test/groovy/org/transmartproject/rest/RestExportServiceSpec.groovy

```diff
@@ -0,0 +1,27 @@
+package org.transmartproject.rest
+
+import grails.test.mixin.TestFor
+import org.transmartproject.rest.dataExport.RestExportService
+import spock.lang.Specification
+
+/**
+ * See the API for {@link grails.test.mixin.services.ServiceUnitTestMixin} for usage instructions
+ */
+@TestFor(RestExportService)
+class RestExportServiceSpec extends Specification {
+//
+//    @Autowired
+//    RestDataExportService restDataExportService
+    
+    def setup() {
+    }
+
+    def cleanup() {
+    }
+
+    void "test something"() {
```

Comment:
This test was auto-generated by Grails. I didn't remove it, because I am not sure if we need tests for RestExportService, they will partially duplicate e2e tests.